mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -310,7 +310,7 @@ class KMeans @Since("1.5.0") (
     fit(dataset, handlePersistence)
   }
 
-  @Since("2.2.0")
+  @Since("2.1.0")
```

Comment:
This method doesn't appear to be in 2.1 though:
https://github.com/apache/spark/blob/branch-2.1/mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

let's close this please